backend/Ballerina-Runner/src/main/java/org/wso2/msf4j/ballerina/Application.java

```diff
@@ -52,7 +52,7 @@
 
     // Extract the Ballerina distribution and copy psfinder script.
     static {
-        String DISTRIBUTION_ZIP = "ballerina-0.87.zip";
+        String DISTRIBUTION_ZIP = "ballerina-0.92.zip";
```

Comment:
@muthulee , For the composer we have automated the version upgrades using google's replacer plugin.

https://github.com/ballerinalang/composer/blob/master/modules/web/pom.xml#L155

But the problem is once the version upgrade is done, someone has to push the changes as the changes are done by the maven commands.